core/templates/components/ck-editor-helpers/ck-editor-4-rte.directive.ts

```diff
@@ -66,6 +66,8 @@ angular.module('oppia').directive('ckEditor4Rte', [
         title: false,
         floatSpaceDockedOffsetY: 15,
         extraAllowedContent: extraAllowedContentRules,
+        forcePasteAsPlainText: true,
+
```

Comment:
Remove extra new line (it's not required, right?)

Reply:
done!